tutorials/physics/ray-casting.rst

```diff
@@ -151,25 +151,22 @@ data:
        metadata: Variant() # metadata of collider
     }
 
-The data is similar in 3D space, using Vector3 coordinates. Note that to enable collisions
-with Area3D, the boolean parameter ``collide_with_areas`` must be set to ``true``.
+The data is similar in 3D space, using Vector3 coordinates. Here an example assuming that `$Camera3D` is the main camera node:
 
 .. tabs::
  .. code-tab:: gdscript GDScript
 
         const RAY_LENGTH = 1000
 
         func _physics_process(delta):
-            var space_state = get_world_3d().direct_space_state
-            var cam = $Camera3D
-            var mousepos = get_viewport().get_mouse_position()
-
-            var origin = cam.project_ray_origin(mousepos)
-            var end = origin + cam.project_ray_normal(mousepos) * RAY_LENGTH
-            var query = PhysicsRayQueryParameters3D.create(origin, end)
-            query.collide_with_areas = true
-
-            var result = space_state.intersect_ray(query)
+        	var params = PhysicsRayQueryParameters3D.new()
+        	params.from = $Camera3D.origin
+        	params.to = $Camera3D.project_position(
+        							get_viewport().get_mouse_position(), 
+        							RAY_LENGTH )
+        	var result = get_world_3d().direct_space_state.intersect_ray(params)
```

Comment:
```suggestion
            var params = PhysicsRayQueryParameters3D.new()
            params.from = $Camera3D.origin
            params.to = $Camera3D.project_position(
                    get_viewport().get_mouse_position(),
                    RAY_LENGTH)
            var result = get_world_3d().direct_space_state.intersect_ray(params)
```
Two indents for continuations, and use four spaces for indentation, alternatively:
```suggestion
            var params = PhysicsRayQueryParameters3D.new()
            params.from = $Camera3D.origin
            params.to = $Camera3D.project_position(get_viewport().get_mouse_position(), RAY_LENGTH)
            var result = get_world_3d().direct_space_state.intersect_ray(params)
```

Reply:
if possible I like lines not to be too long.